apify-docs-theme/src/config.js

```diff
@@ -241,8 +241,8 @@ const themeConfig = ({
     },
     algolia: {
         appId: 'N8EOCSBQGH',
-        apiKey: 'b43e67a96ed18c7f63f5fd965906a96d', // search only (public) API key
-        indexName: 'apify_sdk',
+        apiKey: 'e97714a64e2b4b8b8fe0b01cd8592870', // search only (public) API key
+        indexName: 'crawler_dev_apify_sdk',
```

Comment:
so we will keep the new index?

Reply:
That's definitely something to think about - while Algolia versions the Crawler configs, it's IMO a bit opaque / hidden there. I don't really have any preference regarding this - I did it like this since it's a tad easier to revert from here in case of any problems.

Reply:
yeah sure, i was just curious if this is to be changed or final. i guess there is not much to review here, so let's merge and see it in the wild? worst case we can revert

Reply:
indeed, I just need the approving review to be able to merge :)